{"name": "Проверка вывода ошибки при авторизации курьера с не верными логином и паролем", "status": "passed", "steps": [{"name": "Отправляем POST запрос на ручку /api/v1/courier/login", "status": "passed", "parameters": [{"name": "payloads", "value": "{'login': 'CdgBgyA', 'password': '123141'}"}], "start": 1733208103548, "stop": 1733208103878}, {"name": "Проверяем статус код", "status": "passed", "parameters": [{"name": "code", "value": "404"}], "start": 1733208103878, "stop": 1733208103878}, {"name": "Проверяем сообщение об ошибке Учетная запись не найдена", "status": "passed", "steps": [{"name": "Получаем сообщение об ошибке", "status": "passed", "start": 1733208103878, "stop": 1733208103878}], "start": 1733208103878, "stop": 1733208103878}], "attachments": [{"name": "stdout", "source": "ee34d7e0-d3d3-4020-95cf-15a19ba9e624-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "params", "value": "'login'"}], "start": 1733208103548, "stop": 1733208103878, "uuid": "f7840c95-9d7d-405c-81d4-39ee82832bd0", "historyId": "18e720dff1205c33456ff2fa46714d0f", "testCaseId": "fdba0aa5c612fb0c6fd040949622be8b", "fullName": "tests.test_login_courier.TestLoginCourier#test_invalid_login_or_password_returns_error", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-40MQLH4"}, {"name": "thread", "value": "6356-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}